components/eam/src/physics/cam/cldera_dynamic_tracers.F90

```diff
@@ -26,18 +26,28 @@ module cldera_dynamic_tracers
   ! Public interfaces
   public :: cldera_dynamic_tracers_register        ! register constituents
   public :: cldera_dynamic_tracers_implements_cnst ! true if constituent is implemented by this package
+  public :: cldera_dynamic_tracers_is_pt ! true if constituent is implemented by this package
+  public :: cldera_dynamic_tracers_is_pv ! true if constituent is implemented by this package
+  public :: cldera_dynamic_tracers_is_enabled ! true if constituent is implemented by this package
   public :: cldera_dynamic_tracers_init            ! initialize history fields, datasets
   public :: cldera_dynamic_tracers_init_cnst       ! initialize constituent field
+  public :: cldera_dynamic_tracers_timestep_tend
   public :: cldera_dynamic_tracers_readnl          ! read namelist options
+  public :: cldera_dynamic_tracers_pv_idx
+  public :: cldera_dynamic_tracers_pt_idx
+  public :: cldera_dynamic_tracers_pv_offset
+
+  real(r8), parameter :: cldera_dynamic_tracers_pv_offset = 1_r8
 
   ! ----- Private module data
   integer, parameter :: ncnst=2  ! number of constituents implemented by this module
 
   ! constituent names
-  character(len=8), parameter :: c_names(ncnst) = (/'PV_TRCR     ', 'PT_TRCR    '/)
+  character(len=8), parameter :: c_names(ncnst) = (/'PV_TRCR_   ', 'PT_TRCR    '/)
```

Comment:
Why was an underscore added to the PV name here, and why do we need the new variable `pv_out_name`? Why is this treated differently than the PT?